riff-raff/app/controllers/ContinuousDeployController.scala

```diff
@@ -1,20 +1,20 @@
 package controllers
 
-import play.api.mvc.Controller
+import play.api.mvc.{AnyContent, BaseController, ControllerComponents}
 import play.api.data.Forms._
 import play.api.data.Form
-import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.i18n.I18nSupport
 import play.api.libs.ws.WSClient
-
 import java.util.UUID
 
 import org.joda.time.DateTime
-
 import ci.{ContinuousDeploymentConfig, Trigger}
+import com.gu.googleauth.AuthAction
 import persistence.ContinuousDeploymentConfigRepository
 import resources.PrismLookup
 
-class ContinuousDeployController(prismLookup: PrismLookup)(implicit val messagesApi: MessagesApi, val wsClient: WSClient) extends Controller with Logging with LoginActions with I18nSupport {
+class ContinuousDeployController(prismLookup: PrismLookup, authAction: AuthAction[AnyContent], val controllerComponents: ControllerComponents)(implicit val wsClient: WSClient)
+  extends BaseController with Logging with I18nSupport {
```

Comment:
@DiegoVazquezNanini asked about this in the play-googleauth PR. My conclusion was that the difference was minimal (https://github.com/playframework/playframework/blob/master/framework/src/play/src/main/scala/play/api/mvc/Controller.scala#L168) and that this avoided a tiny bit of repetition, so was marginally preferable.